cmd/snowflake/root.go

```diff
@@ -172,7 +172,7 @@ func NewSnowflakeCmd() *cobra.Command {
 
 		// 2. create changefeed
 		if mode == RunModeFull || (mode == RunModeIncrementalOnly && sindURIStr == "") {
-			increS3StoragePath, err := url.JoinPath(s3StoragePath, "increment")
+			increStoragePath, err := url.JoinPath(storagePath, "increment")
```

Comment:
I think we could use storagePath instead of increS3StoragePath since we need to support gcs and perhaps azure in the future.